<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestaoProduto</class>
 <widget class="QDialog" name="GestaoProduto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GESTÃO DE PRODUTO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/img/img/produto.png</normaloff>:/img/img/produto.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidgetGestaoProduto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGPNovo">
    <attribute name="title">
     <string>Novo</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>951</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layoutGridGPN">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="layoutGPNCodigo">
          <item>
           <widget class="QLabel" name="labelGPNCodigo">
            <property name="text">
             <string>Código</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGPNCodigo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutGPNQuantidade">
          <item>
           <widget class="QLabel" name="labelGPNQuantidade">
            <property name="text">
             <string>Quantidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGPNQuantidade"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutGPNPreco">
          <item>
           <widget class="QLabel" name="labelGPNPreco">
            <property name="text">
             <string>Preço</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGPNPreco"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="layoutGPNCadastrar">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGPNCadastrar">
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelGPNDescricao">
          <property name="text">
           <string>Descrição</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditGPNDescricao"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>153</y>
       <width>490</width>
       <height>10</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>490</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>3</width>
       <height>150</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>3</width>
       <height>150</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabGPAlterar">
    <attribute name="title">
     <string>Alterar </string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>951</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layoutGridGPA">
      <item row="2" column="1">
       <spacer name="spacerGPADD">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="layoutGPABuscarAlterar">
        <item>
         <spacer name="spacerGPAE">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGPABuscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGPAAlterar">
          <property name="text">
           <string>Alterar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerGPAD">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="layoutGPADescricao">
        <item>
         <widget class="QLabel" name="labelGPADescricao">
          <property name="text">
           <string>Descrição</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditGPADescricao"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="layoutGPACodigo">
          <item>
           <widget class="QLabel" name="labelGPACodigo">
            <property name="text">
             <string>Código</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGPACodigo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutGPAQuantidade">
          <item>
           <widget class="QLabel" name="labelGPAQuantiade">
            <property name="text">
             <string>Quantidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGPAQuantidade"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layoutGPAPreco">
          <item>
           <widget class="QLabel" name="labelGPAPreco">
            <property name="text">
             <string>Preço</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGPAPreco"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>3</width>
       <height>150</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>490</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>3</width>
       <height>150</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>157</y>
       <width>490</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabGPGestao">
    <attribute name="title">
     <string>Gestão</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutGPGCodigo">
      <item>
       <widget class="QLabel" name="labelGPGCodigo">
        <property name="text">
         <string>Código</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditGPGCodigo"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonGPGExcluir">
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>3</width>
       <height>90</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_10">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>16</width>
       <height>90</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>210</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>210</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>3</width>
       <height>510</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_15">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>130</y>
       <width>3</width>
       <height>510</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>940</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>640</y>
       <width>940</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>921</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonGPGListar">
          <property name="text">
           <string>Listar</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetGPGLista">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_14">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>line_14</zorder>
     <zorder>listWidgetGPGLista</zorder>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>360</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>img/produto.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>layoutWidget_2</zorder>
    <zorder>line_9</zorder>
    <zorder>line_10</zorder>
    <zorder>line_11</zorder>
    <zorder>line_12</zorder>
    <zorder>line_13</zorder>
    <zorder>line_15</zorder>
    <zorder>line_17</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>line_16</zorder>
    <zorder>label</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>15</y>
     <width>1031</width>
     <height>671</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Recursos.qrc">:/img/img/pucpb.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidgetGestaoProduto</tabstop>
  <tabstop>lineEditGPNCodigo</tabstop>
  <tabstop>lineEditGPNQuantidade</tabstop>
  <tabstop>lineEditGPNPreco</tabstop>
  <tabstop>lineEditGPNDescricao</tabstop>
  <tabstop>pushButtonGPNCadastrar</tabstop>
  <tabstop>lineEditGPACodigo</tabstop>
  <tabstop>lineEditGPAQuantidade</tabstop>
  <tabstop>lineEditGPAPreco</tabstop>
  <tabstop>lineEditGPADescricao</tabstop>
  <tabstop>pushButtonGPABuscar</tabstop>
  <tabstop>pushButtonGPAAlterar</tabstop>
  <tabstop>lineEditGPGCodigo</tabstop>
  <tabstop>pushButtonGPGExcluir</tabstop>
  <tabstop>pushButtonGPGListar</tabstop>
  <tabstop>listWidgetGPGLista</tabstop>
 </tabstops>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
